modules/imgcodecs/include/opencv2/imgcodecs.hpp

```diff
@@ -96,6 +96,7 @@ enum ImwriteFlags {
        IMWRITE_PNG_COMPRESSION     = 16, //!< For PNG, it can be the compression level from 0 to 9. A higher value means a smaller size and longer compression time. If specified, strategy is changed to IMWRITE_PNG_STRATEGY_DEFAULT (Z_DEFAULT_STRATEGY). Default value is 1 (best speed setting).
        IMWRITE_PNG_STRATEGY        = 17, //!< One of cv::ImwritePNGFlags, default is IMWRITE_PNG_STRATEGY_RLE.
        IMWRITE_PNG_BILEVEL         = 18, //!< Binary level PNG, 0 or 1, default is 0.
+       IMWRITE_PNG_FILTER          = 19, //!< One of cv::ImwritePNGFilterFlags, default is IMWRITE_PNG_FILTER_SUB.
```

Comment:
Are there any changes against 4.11 (APNG support) and 4.10 (old style PNG)?

Reply:
It'll be great to add some accuracy test for the new option. You can, for example, compare file sizes or byte array sizes to ensure the compression applied properly.

Reply:
> It'll be great to add some accuracy test for the new option. You can, for example, compare file sizes or byte array sizes to ensure the compression applied properly.

as far as i tested file sizes are various according used zlib or zlib-ng  ( maybe also used versions creates different sizes)

Reply:
Let's compare fast/best compression to ensure best is smaller the fast, for example.

Reply:
> Are there any changes against 4.11 (APNG support) and 4.10 (old style PNG)?

default values are same as 4.10